### PR TITLE
Simplify Quickstarts

### PR DESCRIPTION
**Pull request by kraftp**

### CODE REVIEW

File: docs/getting-started/index.md [hunk 1]
```diff
@@ -1,14 +1,9 @@
 # Getting Started
 
-Welcome to DBOS! To get started, follow these tutorials to create and deploy your first DBOS application in minutes:
+Welcome to DBOS! Here's how you should get started:
```
Comment (maxdml) [inline docs/getting-started/index.md:3]: `Here's the easiest path to get started` -> sounds less commanding

File: docs/getting-started/quickstart.md [hunk 2]
```diff
@@ -89,10 +90,9 @@ This creates a folder for your application, configures its layout, and installs
 If successful, it should print `Application initialized successfully!`.
 By default, `init` instantiates a "Hello, Database!" application which greets users and tracks the count of greetings per user.
 
+## Running Locally
 
-### Getting Started
-
-Before you can launch your app, you need a database.
+Before you can run your app, you need a database.
```
Comment (maxdml) [inline docs/getting-started/quickstart.md:95]: At the end of this section (or at the beginning), I would be clear you can run the SDK anywhere.

**Review by maxdml — APPROVED** [review]

File: docs/getting-started/quickstart.md [hunk 3]
```diff
@@ -5,13 +5,14 @@ sidebar_position: 1
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# DBOS SDK Quickstart
+# DBOS Quickstart
 
 Here's how to get a simple DBOS "Hello, Database!" application up and running in less than five minutes.
+First we'll show you how to run it locally, then we'll show you how to deploy it to DBOS Cloud!
```
Comment (qianl15) [inline docs/getting-started/quickstart.md:11]:
```suggestion
First we'll show you how to run it locally, then we'll show you how to deploy it serverlessly to DBOS Cloud!
```

File: docs/getting-started/quickstart.md [hunk 4]
```diff
@@ -5,13 +5,14 @@ sidebar_position: 1
...
 
-### System Requirements
+## System Requirements
```
Comment (qianl15) [inline docs/getting-started/quickstart.md:13]: Maybe the title is just `Installation`, and we say we support macOS, Windows (including WSL), and Linux.
https://nextjs.org/docs/getting-started/installation

File: docs/cloud-tutorials/database-management.md [hunk 5]
```diff
@@ -24,22 +24,20 @@ npx dbos-cloud login
 
 Before you can deploy an application to DBOS Cloud, you must provision a Postgres database instance for it.
 You must choose a database instance name, username and password.
-Both the database instance name and username should be between 3 and 30 characters and contain only lowercase letters and numbers, dashes (`-`), and underscores (`_`).
+Both the database instance name and username must be 3 to 16 characters long and contain only lowercase letters, numbers and underscores.
 The database password must contain 8 or more characters.
 Run this command and choose your database password when prompted (it should take ~5 minutes to provision):
```
Comment (qianl15) [inline docs/cloud-tutorials/database-management.md:29]:
```suggestion
Run this command and choose your database password when prompted (it should take up to 5 minutes to provision):
```

**Review by qianl15 — APPROVED** [review]

**Pull request merged**